changelog.md

```diff
@@ -0,0 +1,40 @@
+# Changelog
+All notable changes to this project will be documented in this file.  
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
+and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+
+## [1.0](https://github.com/MariaMaximova/convert_temp/tree/v1.0) - 2022-August-06
+### Added
+- Readme and description were added.
+- Create some interactivity.
+- Added ability to work with CLI.
+
+### Fixed
+- Some grammatical issues and typos
+
+
+## [1.0.1](https://github.com/MariaMaximova/convert_temp/tree/v1.0.1) - 2022-Aug-10
+### Added
+- *setup.py* file.
+- Added separate folder *converter* within file *converter.py* with all convertion logic.
+
+### Removed
+- Convert logic in the main.py file.
+
+### Fixed
+- Some grammatical issues and typos.
+
+
+## [1.0.2] - 2022 - August - 
```

Comment:
@MariaMaximova, considering there is no 1.0.2 version yet, I suggest you removing version and adding `## Unreleased` section here.

Reply:
Section [Unreleased](https://github.com/MariaMaximova/convert_temp/pull/9/commits/1ab8b94e86226a891571c2a8960d6bb6b781e07d#diff-3bd14d078188074c410028847113ceae68865d0ad5b844a27183ef87fbe2fcc3) was added